dymos/phase/test/test_analytic_phase.py

```diff
@@ -202,6 +202,42 @@ def compute_partials(self, inputs, partials):
 
         assert_near_equal(y, expected(t))
 
+    def test_analytic_phase_load_case(self):
+
+        p = om.Problem()
+        traj = p.model.add_subsystem('traj', dm.Trajectory())
+
+        phase = dm.AnalyticPhase(ode_class=SimpleIVPSolution, num_nodes=10)
+
+        traj.add_phase('phase', phase)
+
+        phase.set_time_options(units='s', targets=['t'], fix_initial=True, fix_duration=True)
+        phase.add_state('y')
+        phase.add_parameter('y0', opt=False)
+
+        p.setup()
+
+        p.set_val('traj.phase.t_initial', 0.0, units='s')
+        p.set_val('traj.phase.t_duration', 2.0, units='s')
+        p.set_val('traj.phase.parameters:y0', 0.5, units='unitless')
+
+        dm.run_problem(p, simulate=False)
+
+        # Change the inputs so that we get different answers unless we reload the same case.
+        p.set_val('traj.phase.t_initial', 0.0, units='s')
+        p.set_val('traj.phase.t_duration', 1.0, units='s')
+        p.set_val('traj.phase.parameters:y0', 0.6, units='unitless')
+
+        # Load the previous case and rerun
+        dm.run_problem(p, simulate=False, restart='dymos_solution.db')
```

Comment:
Naive question; do we need to use temp_dirs or anything here, given that we're reading in the `dymos_solution.db` directly? Would this ever clash with other tests or is this not a concern?

Reply:
That seems like a good idea.

Reply:
Good catch